resources/views/processes/scripts/list.blade.php

```diff
@@ -26,159 +26,86 @@ class="fas fa-plus"></i>
+                        <div class="invalid-feedback" v-for="run_as_user_id in addScript.addError.run_as_user_id"> @{{run_as_user_id}}</div>
+                    </div>
+                    <div class="form-group">
+                        {!! Form::label('timeout', __('Timeout')) !!}
+                        <div class="form-row ml-0">
+                            {!! Form::text('timeout', null, ['id' => 'timeout', 'class'=> 'form-control col-2',
+                            'v-model' => 'addScript.timeout', 'pattern' => '[0-9]*', 'v-bind:class' => '{"form-control":true, "is-invalid":addScript.addError.timeout}']) !!}
+                            {!! Form::range(null, null, ['id' => 'timeout-range', 'class'=> 'custom-range col ml-1 mt-2',
+                            'v-model' => 'addScript.timeout', 'min' => 0, 'max' => 300]) !!}
+                            <div class="invalid-feedback" v-for="timeout in addScript.addError.timeout">@{{timeout}}</div>
                         </div>
-                    @endif
-                    <div class="modal-footer">
-                        <button type="button" class="btn btn-outline-secondary" data-dismiss="modal" @click="onClose">
-                            {{__('Cancel')}}
-                        </button>
-                        @if ($config->countCategories !== 0)
-                            <button type="button" class="btn btn-secondary ml-2" @click="onSubmit" :disabled="disabled">
-                                {{__('Save')}}
-                            </button>
-                        @endif
+                        <small class="form-text text-muted" v-if="!addScript.addError.timeout">
+                            {{ __('Enter how many seconds the Script runs before timing out (0 is unlimited).') }}
+                        </small>
                     </div>
-                </div>
-            </div>
-        </div>
-    @endcan
+                @else
+                    <div>{{__('Categories are required to create a script')}}</div>
+                    <a href="{{ url('designer/scripts/categories') }}" class="btn btn-primary container mt-2">
+                        {{__('Add Category')}}
+                    </a>
+                @endif
+                <template v-slot:modal-footer>
+                    <button type="button" class="btn btn-outline-secondary" @click="$refs.addScript.hide()">
+                        {{__('Cancel')}}
+                    </button>
+                    @if ($config->countCategories !== 0)
+                        <button type="button" class="btn btn-secondary ml-2" @click="onSubmit">
+                            {{__('Save')}}
+                        </button>
+                    @endif
+                </template>
+            </b-modal>
+        @endcan
+    </div>
 
 @section('js')
     <script src="{{mix('js/processes/scripts/index.js')}}"></script>
-
-    @can('create-scripts')
```

Comment:
Please put this logic back. The point, while not as compact, is to make it easier to work with by seeing the Vue object via the blade and not having to hunt for it.